### PR TITLE
March 23 Release Notes

### PR DESCRIPTION
**Pull request by ncphillips**
- [x] Scott's theming post/docs released
- [x] Add sentence or two communicating a rough idea about what the core-team is working on this week

### CODE REVIEW

File: content/blog/2020-03-23-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Since most of the development for Open Authoring was done directly in the [tinacms.org repository](https://github.com/tinacms/tinacms.org "GitHub: tinacms.org") itself some house keeping is in order. We changed the [theming system](/blog/designing-an-extensible-styling-system) to be more compatible with inline editing, started moving over our new blocks, and started refactoring `tinacms` to better accommodate the new [Toolbar UI](https://tinacms.org/blog/tinacms-ui-whats-next "TinaCMS UI: What's Next?"). To help make this process smoother, Jeff created the [**@tinacms/webpack-helpers**](https://github.com/tinacms/tinacms/tree/master/packages/@tinacms/webpack-helpers "@tinacms/webpack-helpers") package to make it easy to test changes to tinacms from outside the monorepo.
+
+This week the TinaCMS core team will continue down the path towards Open Authoring for everyone. In order to get a better idea of what packages and guides we will be doing some refactoring in tinacms.org and looking into how the Toolbar UI will relate to the Sidebar. We're also exploring ways to introduce a versioned API reference for all the `tinacms/tinacms` packages.
```
Comment (dwalkr) [inline content/blog/2020-03-23-release-notes.md]: there's like, a word missing or something in the first sentence here

File: content/blog/2020-03-23-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+### Enhancements
+
+#### **@tinacms/styles**
+
+* theming is done using CSS custom properties
+
+#### **react-tinacms-inline**
+
+* **Improved Blocks UI:** See the [PR for more details](https://github.com/tinacms/tinacms/pull/908 "Inline Block Fixes")
+* **New Component:** `BlockImage` let's users add images to blocks.
```
Comment (dwalkr) [inline content/blog/2020-03-23-release-notes.md]:
```suggestion
* **New Component:** `BlockImage` lets users add images to blocks.
```

File: content/blog/2020-03-23-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Linking apps to a monorepo can be tricky. Tools like `npm link` are buggy and introduce inconsistencies with module resolution. If multiple modules rely on the same package you can easily end up with multiple instances of that package, this is problematic for packages like `react` which expect only one instance. If your app uses `webpack` you can get around these issues by ensuring that your dependencies come from a specific path on your system. This is especially helpful for working on the TinaCMS monorepo while using its packages in your app. `@tinacms/webpack-helpers` makes it easy to set up.
+
+[Checkout the README](https://github.com/tinacms/tinacms/tree/master/packages/@tinacms/webpack-helpers "@tinacms/webpack-helpers") to learn how to develop TinaCMS with your website.
+
+## Contributors
+
+Thanks to everyone for contributing.
+
+| # Commits | Name |
```
Comment (dwalkr) [inline content/blog/2020-03-23-release-notes.md]: It seems weird to have you, me and Jyoti referenced here by our github handles when everyone else has their full name. I assume this is auto-generated?

Comment (ncphillips) [reply content/blog/2020-03-23-release-notes.md]: It's from the git log yeah

Comment (ncphillips) [reply content/blog/2020-03-23-release-notes.md]: Do you want your full name?

```
Donald J. Walker
```

File: content/blog/2020-03-23-release-notes.md (hunk 2 above)
Comment (dwalkr) [reply content/blog/2020-03-23-release-notes.md]: @ncphillips looks like Tina changed this back

File: content/blog/2020-03-23-release-notes.md (hunk 3 above)
Comment (dwalkr) [reply content/blog/2020-03-23-release-notes.md]: that ain't even right so nah

**Pull request merged**